.appveyor.yml

```diff
@@ -22,38 +22,28 @@ environment:
 
     # See: https://www.appveyor.com/docs/installed-software/#python
 
-    - PYTHON_ARCH: "32"
-      NOX_SESSION: "unit(py='2.7')"
-
-    - PYTHON_ARCH: "64"
-      NOX_SESSION: "unit(py='2.7')"
-
-    - PYTHON_ARCH: "32"
-      NOX_SESSION: "unit(py='3.5')"
-
-    - PYTHON_ARCH: "64"
-      NOX_SESSION: "unit(py='3.5')"
-
-    - PYTHON_ARCH: "32"
-      NOX_SESSION: "unit(py='3.6')"
-
-    - PYTHON_ARCH: "64"
-      NOX_SESSION: "unit(py='3.6')"
-
-    - PYTHON_ARCH: "32"
-      NOX_SESSION: "unit(py='3.7')"
-
-    - PYTHON_ARCH: "64"
-      NOX_SESSION: "cover"
-
-    - PYTHON_ARCH: "64"
-      NOX_SESSION: "functional(py='3.7')"
-
-    - PYTHON_ARCH: "64"
-      NOX_SESSION: "doctest"
-
-    - PYTHON_ARCH: "64"
-      NOX_SESSION: "check_journal(machine='appveyor')"
+    - NOX_SESSION: "unit-2.7-32"
+      PYTHON_ARCH: "32"
```

Comment:
`PYTHON_ARCH` is kept around as cruft to signal if `MINGW_32` or `MINGW_64` is used